terraform/projects/pr-dashboard/rds.tf

```diff
@@ -0,0 +1,88 @@
+
+resource "aws_db_instance" "rds" {
+  identifier = "web-platform-tests-db"
+  storage_type = "gp2"
+  allocated_storage = 10
+  engine = "postgres"
+  engine_version = "9.5.4"
+  instance_class = "db.t2.small"
+  name = "wptdash"
+  username = "wptdash"
+  password = "${trimspace(file("${path.module}/secrets/wpt-db-password.txt"))}"
+  multi_az = true
+  backup_retention_period = 30
+  backup_window = "05:00-18:30"
+  maintenance_window = "sun:03:00-sun:04:00"
+  allow_major_version_upgrade = true
+  vpc_security_group_ids = [
+    "${aws_security_group.web-platform-tests-db1.id}",
+  ]
+}
+
+
+resource "aws_security_group" "web-platform-tests-db1" {  
+  name = "web-platform-tests-db1"
+
+  description = "RDS postgres servers (terraform-managed)"
+  vpc_id = "${module.vpc.id}"
+
+  # Only postgres in
+  ingress {
+    from_port = 5432
+    to_port = 5432
+    protocol = "tcp"
+    cidr_blocks = [
+      "${var.subnet_cidr_blocks}",
```

Comment:
Six of this half a dozen of the other, but you may wish to use `${module.subnet.cider_blocks}`.